hgext/set_landing_system.py

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
Wondering if it makes sense to expand this `hgext` to be `landoext` and put the hooks in there? Not sure we'll add any more Mercurial customizations or not.